Formula/fhir-validation-cli.rb

```diff
@@ -0,0 +1,21 @@
+class FhirValidationCli < Formula
+  desc "FHIR command-line validator"
+  homepage "https://wiki.hl7.org/Using_the_FHIR_Validator"
+  url "https://fhir.github.io/latest-ig-publisher/org.hl7.fhir.validator.jar"
+  version "4.1.40-SNAPSHOT"
+  sha256 "350ac5a1b258946828fa5ad0bc3784bad427577336bab9a60c382b5470c60a1c"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
+
+  def install
+    inreplace "fhir-validation-cli", /SCRIPTDIR=(.*)/, "SCRIPTDIR=#{libexec}"
+    libexec.install "fhir-validation-cli.jar"
+    bin.install "fhir-validation-cli"
+  end
+
+  test do
+    system bin/"fhir-validation-cli"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.